msu/utils/array.nut

```diff
@@ -55,6 +55,17 @@
 		if (idx != null) return _array.remove(idx);
 	}
 
+	function removeValues( _array, _values )
+	{
+		for (local i = _array.len() - 1; i >= 0; i--)
+		{
+			if (_values.find(_array[i]) != null)
+			{
+				_array.remove(i);
+			}
+		}
+	}
```

Comment:
You should still return the array, its always annoying to remember which function does and which doesn't.

Reply:
Is it better to return the array itself or a new array containing all the values that were removed?

Reply:
Returning the removed values is not consistent with other languages imo, if you need that functionality its very easy to write it yourself

Reply:
We discussed and think this would be non-standard so no